File: DoAnMau/app/src/main/java/com/example/phuongnamlib/Models/AuthorManager.java
package com.example.phuongnamlib.Models;

public class AuthorManager {
    int idAuthor;
    int idBook;

    public AuthorManager(int idAuthor, int idBook) {
        this.idAuthor = idAuthor;
        this.idBook = idBook;
    }

    public int getIdAuthor() {
        return idAuthor;
    }

    public void setIdAuthor(int idAuthor) {
        this.idAuthor = idAuthor;
    }

    public int getIdBook() {
        return idBook;
    }

    public void setIdBook(int idBook) {
        this.idBook = idBook;
    }
}

File: DoAnMau/pnlib/pnlib/Models/AuthorManager.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace pnlib.Models
{
    public partial class AuthorManager
    {
        public int IdAuthor { get; set; }
        public int IdBook { get; set; }

        public virtual Author IdAuthorNavigation { get; set; }
        public virtual Book IdBookNavigation { get; set; }
    }
}
